src/pages/management/configuration-api/v3.6/index.mdx

```diff
@@ -1853,6 +1854,543 @@ curl -X POST \
+        "id": 123,
+        "active": false,
+        "name": "Updated welcome greeting",
+        "rules": [
+          {
+            "condition": "and",
+            "type": "url_current",
+            "operator": "contains",
+            "value": "product"
+          }
+        ],
+        "properties": {
+          "addon": "chat_widget",
+          "is_exit_intent": "true"
+        },
+        "rich_message": {
+          "template_id": "text",
+          "elements": [
+            {
+              "title": "Updated greeting message!",
+              "subtitle": "Can we help you find a product?"
+            }
+          ]
+        }
+      }'
+```
+
+</CodeSample>
+
+</Code>
+
+</Section>
+
+<Section>
+
+<Text>
+
+### Delete Greeting
+
+Deletes a greeting specified by `id`.
+
+#### Specifics
+
+|                                  |                                                                         |
+| -------------------------------- | ----------------------------------------------------------------------- |
+| Method URL                       | `https://api.livechatinc.com/v3.6/configuration/action/delete_greeting` |
+| Required scopes                  | `greetings:rw`                                                          |
+| [Batch support](#batch-requests) | No                                                                      |
+
+#### Request
+
+| Parameter | Data type | Required | Notes                          |
+| --------- | --------- | -------- | ------------------------------ |
+| `id`      | `int`     | Yes      | ID of the greeting to delete   |
+
+#### Response
+
+No response payload (`200 OK`).
+
+</Text>
+
+<Code>
+
+<CodeSample path={'REQUEST'}>
+
+```shell
+curl -X POST \
+  https://api.livechatinc.com/v3.6/configuration/action/delete_greeting \
+  -H 'Authorization: Bearer <your_access_token>' \
+  -H 'Content-Type: application/json' \
+  -d '{
+        "id": 123
+      }'
+```
+
+</CodeSample>
+
+</Code>
+
+</Section>
+
+<Section>
+
+<Text>
+
+### List Greetings
+
+Returns a list of greetings, optionally filtered by groups. The method supports pagination to handle large result sets.
+
+#### Specifics
+
+|                                  |                                                                        |
+| -------------------------------- | ---------------------------------------------------------------------- |
+| Method URL                       | `https://api.livechatinc.com/v3.6/configuration/action/list_greetings` |
+| Required scopes                  | `greetings:ro`                                                         |
+| [Batch support](#batch-requests) | No                                                                     |
+
+#### Request
+
+| Parameter  | Data type | Required | Default | Notes                                                                                                                                                                                         |
```

Comment:
We don't have a 'default' column in our docs' tables, if there are any default values, they should be added in the notes